src/views/Splash.vue

```diff
@@ -14,7 +14,7 @@
             : { path: '/onboarding/home', query: { isImport: true } }
         "
         ><p class="text-center" id="import_with_seed_phrase_option">
-          Import with seed phrase
+          Forgot password? Import with seed phrase
```

Comment:
If I just installed the wallet. It still say's "Forgot password?" But it shouldn't

Reply:
I had that implemented but the design shows this:

<img width="659" alt="Screenshot 2022-01-18 at 15 43 21" src="https://user-images.githubusercontent.com/47292405/149931310-aa8eb9ca-83cf-458a-9378-32244f2a7a24.png">

Reply:
i mean that's surely wrong? 😄  There is no password at that stage.

Reply:
I agree. I have changed it to just "Import with seed phrase" now.